kotlinx-coroutines-core/common/src/flow/operators/Lint.kt

```diff
@@ -186,3 +186,15 @@ public suspend inline fun <T> SharedFlow<T>.toSet(destination: MutableSet<T>): N
 @InlineOnly
 public suspend inline fun <T> SharedFlow<T>.count(): Int =
     (this as Flow<T>).count()
+
+/**
+ * @suppress
+ */
+@Suppress("DeprecatedCallableAddReplaceWith")
+@Deprecated(
+    level = DeprecationLevel.WARNING,
```

Comment:
Good call making this a `WARNING` and not an `ERROR`. Wouldn't be nice to break the compilation of someone's code using `last()` instead of `collect()`.